system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.hpp

```diff
@@ -19,22 +19,23 @@
 #include <chrono>
 #include <string>
 
+#include "rclcpp/rclcpp.hpp"
+#include "metrics_statistics_msgs/msg/metrics_message.hpp"
+
```

Comment:
alphasort or an empty line is missing here.

Reply:
Did alphasort.